### DISCUSSION TITLE
Ha su Qnap

### DISCUSSION THREAD

**Discussion by merovese** [indomus/forum#1856] [HUB: Home Assistant]
Ciao a tutti, dopo tante peripezie ho installato Ha su nas Qnap.
Ma da subito ho notato delle differenze: al posto della voce, “configurazione” io ho la voce “impostazioni” ma quella più importante è che non ho la voce “componenti aggiuntivi” qualcuno saprebbe dirmi perché è come poter risolvere? Grazie mille in anticipo

**Comment by snakuzzo** [comment]
Hai letto qui ???

https://indomus.it/focus/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/

**Reply by merovese** [reply]
Si certo, solo che nella ricerca nella "container station" Qnap non c'è questa distinzione

**Reply by snakuzzo** [reply]
Mettiamola così... con container station puoi installare solo core come container docker (leggi [QUI](https://indomus.it/guide/installare-e-configurare-home-assistant-core-con-docker-su-qnap-via-containerstation/)). Non stai installando HA OS. Se vuoi HA OS sul Qnap con i relativi componenti aggiuntivi (addon) devi creare una VM con virtualization station usando l'immagine apposita (leggi [QUI](https://indomus.it/guide/installare-e-configurare-home-assistant-os-su-qnap-via-virtualization-station/))

Tornano all'installazione che hai fatto... Se proprio sei convinto di voler installare Core, l'immagine dovresti prenderla da dockerhub e non da quelle che ti propone di default.

**Reply by merovese** [reply]
Ho seguito il tuo consiglio, l'ho installato con virtualizzazione station, ed ho installato l'os ma ancora qualcosa non va, non ho l dicitura tramite la quale installare hacs che mi permette di installare gli addon che mi occorrono

**Reply by snakuzzo** [reply]
Leggo tanta confusione :(

Hacs non ti fa installare gli addon, ma i custom component... è una cosa diversa!!!
Per installare HACS devi seguire prima una procedura... [QUESTA](https://indomus.it/guide/installare-hacs-home-assistant-community-store-sul-proprio-hub/)

**Reply by merovese** [reply]
Molta confusione, hai ragione

**Reply by merovese** [reply]
Mmmmm, per farla breve, a me serve gestire i device Sonoff ma non riesco ad installare il Sonoff Lan, seguo le guide attentamente ma c'è sempre qualcosa che non va, in questo caso è la repository, ha mi dice che già è installata, quando invece non c'è...

**Reply by snakuzzo** [reply]
se sotto `/config/custom_components/` hai la directory del componente custom, vuol dire che c'è  e devi seguire le istruzioni.
Se c'è lo vedi pure al restart di home assistant nei log...ti dà un warning.
Hai letto la guida ufficiale del custom o la [nostra guida](https://indomus.it/guide/integrare-itead-sonoff-con-firmware-originale-a-home-assistant-v2/) ??

**Reply by merovese** [reply]
Ti ringrazio delle risposte, ma la vedo sempre più dura, un buco nero. Oltre a leggere la guida SONOFF LAN scartabellando sull'app EWeLink ho letto la guida ufficiale per porta EWeLink su ha. La questione rimane ferma sul fatto che io non ho la dicitura "SUPERVISOR". Ho installato ha direttamente sulla virtualization station ma non riesco a seguire le altre guide poiché mi blocco sempre lì

**Reply by snakuzzo** [reply]
Che vuol dire che hai installato ha nella virtualization station? Hai scaricato l'immagine per virtual box e installato seguendo [QUESTA GUIDA](https://indomus.it/guide/installare-e-configurare-home-assistant-os-su-qnap-via-virtualization-station/) ?
La voce di menu relativa al Supervisor la trovi sotto CONFIGURATION.
Ti stai perdendo in un bicchiere d'acqua senza nemmeno l'acqua

**Reply by merovese** [reply]
Buongiorno, si esatto, proprio seguendo la guida che mi hai segnalato, la questione è questa:

EWeLink dice questo:

![image](https://user-images.githubusercontent.com/1671062/187605497-8c181500-1915-4797-a9cb-3acf218d224c.jpeg)

A prescindere dal bicchiere d'acqua, se una qualsiasi guida mi dice di entrare nel menù Supervisor, ed io quel menù non ce l'ho, ovvio che di conseguenza tutto il resto non riesco a farlo, poi tu mi peli di Configuration, immagino tu stia parlando di IMPOSTAZIONI, ma lì di supervisor non c'è traccia

**Reply by snakuzzo** [reply]
Ok...Ho riesumato la VM con HA OS, la voce Supervisor non c'è più (quella guida non è aggiornata)...però permettimi di dire che ci vuole veramente un briciolo di intuito per arrivarci:
Se hai veramente installato HA OS ti basta andare sotto `Settings > Add-ons`  (non Configuration...errore mio)

![bbb](https://user-images.githubusercontent.com/7205371/187619356-55f71b1e-b7e6-4280-9d3e-d44389764edf.png)

Inoltre, non capisco perché tu voglia installare quell'addon al posto del custom component SonoffLAN

**Reply by merovese** [reply]
Permettimi di risponderti (sempre con il dovuto rispetto).
Non siamo alla frutta ma ci siamo appena seduti al tavolo e dovresti averlo capito poiché nel mio primo post ho scritto che non ci capisco nulla dato che è la prima volta che mi avvicino al prodotto; quindi da chi gestisce forum, scrive guide molto dettagliate, mi aspetto siano fatte anche per chi si cimenta per la prima volta con questa realtà.
Comunque sempre grazie per le dritte.

**Reply by indomus** [reply]
> Permettimi di risponderti (sempre con il dovuto rispetto). Non siamo alla frutta ma ci siamo appena seduti al tavolo e dovresti averlo capito poiché nel mio primo post ho scritto che non ci capisco nulla dato che è la prima volta che mi avvicino al prodotto; quindi da chi gestisce forum, scrive guide molto dettagliate, mi aspetto siano fatte anche per chi si cimenta per la prima volta con questa realtà. Comunque sempre grazie per le dritte.

Hai ragione. Prendiamo quanto hai scritto come spunto, se come Diego ha fatto presente c'è un'anomalia sulla guida, vedremo di migliorarla.

**Reply by snakuzzo** [reply]
Sì ma @merovese  io mi riferivo alla guida dell'addon ewelink che hai postato stamattina, non alla guida indomus.

**Reply by merovese** [reply]
Ma credimi la mia non è una critica solo, cerca di metterti nei panni di chi come me, oggi inizia a leggere tante informazioni, contrastanti tra loro, varie versioni tra cui districarsi e altrettante azioni da svolgere per cercare di arrivare alla meta, e credimi non sono proprio un pivellino tecnicamente parlando, e non patente tutto mi trovo spaesato e questa è la motivazione delle domande che ti ho posto. E sempre grazie mille per il tempo che mi hai dedicato, ma vedrai piano piano ce la farò.
@indomus Grazie mille, il tuo semplice messaggio mi è di incoraggiamento. Le guide, come dico sempre io, dovrebbero essere a prova di scimmia, al di sopra di ogni dubbio. Ho molta stima del vostro lavoro!

**Reply by snakuzzo** [reply]
Tutto ok...ma mi trovi in disaccordo sulla guida a prova di scimmia. Le guide uno cerca di farle il più precise possibile, ma devono servire come spunto, non dei manuali di istruzioni

**Comment by duedipicche** [comment]
Sulla guida su "come installare HA su QNAP con container station manca la parte dove mettere la Timezone. Senza quella le integrazioni che hanno al loro interno la richiesta di data e ora hanno come risposta la data e ora nell'orario GMT.Per esempio se io faccio un restart di HA il mio termostato rimette l'ora GMT e non l'ora Italiana.
il comando TZ=Europe/Rome nell'installazione da QNAP Container Station dove devo metterlo? (Container Station non usa il file compose.yam).
Grazie

**Reply by snakuzzo** [reply]
Premesso che la guida è vecchissima e non verrà manutenuta, anche perché non abbiamo più a disposizione il QNAP, puoi passare la TZ come variabile d'ambiente quando crei il container, come indicato [QUI](https://www.qnap.com/it-it/how-to/tutorial/article/come-si-utilizza-container-station-3) sotto **"Ambienti"**

Devi passare la coppia variabile-valore:

Variabile: `TZ`
Valore: `Europe/Rome`

**Reply by duedipicche** [reply]
Grazie per l'attenzione e per la risposta
